### PR TITLE
Ardenian

### LINKED ISSUE #149: Add type to tierlist in RC
Theodotos was asking for an additional piece of information displayed next to the raid tiers.

This is not yet handled by the server, but can be easily added.

### PR DESCRIPTION
**Pull request by Ardenian**
Adds a raid's types to the tier information tab, fixes #149

### CODE REVIEW

File: src/mods/tier.js [hunk 1]
```diff
@@ -190,6 +190,10 @@ idrinth.tier = {
                             content: idrinth.tier.list[list[count]].name
                         },
                         {
+                            type: 'span',
+                            content: idrinth.tier.list[list[count]].types.join( ', ' )
```
Comment (Idrinth) [inline src/mods/tier.js]: I think this is missing a space

**Comment by Ardenian** [comment]
Code style fixed, @Idrinth ( lol, I did not know I can edit the files online, such a great feature)

**Comment by Idrinth** [comment]
would like to know if #219 is fixed in here, any idea, @Ardenian ?

**Comment by w20k** [comment]
LGTM

**Pull request merged**